build/pkgs/mathics/distros/conda.txt

```diff
@@ -1 +0,0 @@
-mathics3
```

Comment:
Instead of removing these files, can we also out-comment (is this English?) them and add a short comment why they are not included? I think this would be helpful for future-us.

Reply:
`bootstrap-conda` uses `./sage package list` with `--no-file distros/conda.txt` to find the packages for which we need to do use `pip`. That's why I removed the conda.txt files.

We can of course just rename the files instead of deleting them.

Reply:
Renaming sounds like a good solution.